explorer/interpreter/type_checker.cpp

```diff
@@ -4401,6 +4426,83 @@ auto TypeChecker::TypeCheckGenericBinding(GenericBinding& binding,
   return Success();
 }
 
+auto GetSize(Nonnull<const Value*> from) -> size_t {
```

Comment:
```suggestion
// Returns the list size for type deduction.
static auto GetSize(Nonnull<const Value*> from) -> size_t {
```

`static` and a function comment.